content/departments/engineering/teams/source/strategy.md

```diff
@@ -1,3 +1,23 @@
-# Source Department Strategy
+# Source Strategy & Mission
 
-TL;DR: The Source department builds value for Sourcegraph by ensuring customers can index all of their code regardless of code host. We believe this is critical for building a long-term defensible value for Sourcegraph as all other code-search solutions are focused on searching across and individual repo or code host.
+## Strategy
+
+The Source team builds value for Sourcegraph by ensuring customers can index and secure all of their code regardless of code host and auth provider.
+
+We believe this is critical for building a long-term defensible value for Sourcegraph as all other code-search solutions are focused on searching across and individual repo or code host.
+
+## Mission
+
+The Source teams is responsible for the syncing, storing, and securing of code in Sourcegraph's code AI platform. We are putting the Source in Sourcegraph.
+
+We are building a foundation on which the rest of Sourcegraph stands. No other part of Sourcegraph can work without source code and properly managed access to it.
+
+Source makes Sourcegraph's code AI platform _universal_, by allowing it to access any code host, through any authentication and authorization provider, that customers require.
```

Comment:
It's nice to say `any authentication and authroization provider, that customers require`, but in reality we need to make hard calls which ones we want to support and which one not. 

Should we maybe reword this to be clear about the expectations here? E.g. something like:
```suggestion
Source makes Sourcegraph's code AI platform _universal_, by allowing it to access any code host, through the majority of authentication and authorization providers that our customers use.
```

Reply:
I reworded it to remove `any`, which is the critical word here, IMHO.